tests/conftest.py

```diff
@@ -774,6 +774,14 @@ def menu_item(menu):
         url='http://example.com/')
 
 
+@pytest.fixture
+def menu_item_list(menu):
+    menu_item_1 = MenuItem.objects.create(menu=menu, name='Link 1')
```

Comment:
https://docs.djangoproject.com/en/dev/ref/models/querysets/#bulk-create

Reply:
Nice. There is probably quite a few places that could use that change! Thanks for letting me know about this new feature!

Reply:
The issue is that the MPTT tree is not yet compatible with bulk creation, it will fail 😕